### DISCUSSION TITLE
blog/jquery-datatable-in-aspnet-core/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#31] [General]
# blog/jquery-datatable-in-aspnet-core/

In this article, we will learn how to use JQuery Datatable in ASP.NET Core with Server Side Processing. We will also be building a simple real-world implementation to help understand JQuery Datatable to it's fullest.

https://codewithmukesh.com/blog/jquery-datatable-in-aspnet-core/

**Comment by karan12k** [comment]
I want to load the table oon  button click on button click I am sending some filter as FromDate and ToDate  I want to send data to API Controller when I hit the controller I receive null in parameter I used normal technique as well as FormData to send data but unable to do it. Can You Help?

**Comment by ekondur** [comment]
Or just use https://www.nuget.org/packages/DatatableJS.Data :)

**Comment by tyariexamki** [comment]
can we do the same using dapper?
